.github/workflows/test-integration-kind-samples.yml

```diff
@@ -72,6 +72,7 @@ jobs:
           make deploy
           kubectl get crds
           kubectl wait --for condition=available --timeout=360s deployment/function-mesh-controller-manager
```

Comment:
Is it only needed to keep "kubectl wait --for condition=ready" here?

Reply:
thanks, addressed in the latest commit